test/dlc_tests/ec2/tensorflow/training/test_tensorflow_training.py

```diff
@@ -269,6 +270,30 @@ def test_tensorflow_addons_cpu(tensorflow_training, ec2_connection, tf2_only, cp
     execute_ec2_training_test(ec2_connection, tensorflow_training, TF_ADDONS_CMD)
 
 
+@pytest.mark.usefixtures("sagemaker")
+@pytest.mark.model("mnist")
+@pytest.mark.integration("tensorflow_io, tensorflow_datasets")
+@pytest.mark.team("frameworks")
+@pytest.mark.parametrize("ec2_instance_type", TF_EC2_SINGLE_GPU_INSTANCE_TYPE, indirect=True)
+def test_tensorflow_io_s3_plugin_gpu(
```

Comment:
is there a way to merge this into an existing test? Adding additional test functions causes horizontal scaling for us